hilti/toolchain/bin/hilti-config.cc

```diff
@@ -126,6 +127,12 @@ int main(int argc, char** argv) {
             continue;
         }
 
+        if ( opt == "--cxx-launcher" ) {
+            if ( auto cxx_launcher = hilti::configuration().cxx_launcher )
+                result.emplace_back(*cxx_launcher);
```

Comment:
```suggestion
                result.emplace_back(*cxx_launcher);
                
```

Reply:
Also made the corresponding change in `spicy-config.cc`.